src/functions/splitNfitTitle.ts

```diff
@@ -2,9 +2,18 @@ import fitty from 'fitty';
 
 import { splitTitle } from './splitTitle';
 
-function fit(titleParts: HTMLSpanElement[]): void {
+interface FittyOptions {
+  maxSize?: number;
+  minSize?: number;
+  multiLine?: boolean;
+  observeMutations?: MutationObserverInit;
+}
+
+function fit(titleParts: HTMLSpanElement[], fittyOptions?: FittyOptions): void {
+  let cleanFittyOptions = Object.fromEntries(Object.entries(fittyOptions).filter(([_, v]) => v != null));
```

Comment:
Hvad sker der helt præcist her?

Reply:
Da fittyoptions kommer fra options og har optionelle keys, som kan være undefined, da de sendes videre uden nogen hensyntagen og jeg gerne vil beholde _fitty_'s defaults renses objectet for properties der er tomme og returneres som et nyt objekt.